v20.2/known-limitations.md

```diff
@@ -194,6 +194,30 @@ Before increasing this value, however, verify that you will not end up saturatin
 
 [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/37906)
 
+### Slow (hung) backups and other operations due to write intent buildup
+
+Transactions do not always clean up their [write intents](architecture/transaction-layer#write-intents) (newly written values) when they are committed or rolled back, due to known bugs. Garbage collection is also rather slow to react to them. This can cause the amount of unresolved write intents to build up over time. While this isn't necessarily a problem in itself, some operations do not handle large amounts of intents well. In particular, backups and other operations that touch large numbers of values may become very slow and appear to hang.
+
+To verify that intents may be causing an issue, visit the DB Console and create a custom chart for the metric `intentcount`. This will show the number of intents present over time. The following query can also be used to get intent counts by range:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM (SELECT start_pretty, end_pretty, range_id, crdb_internal.range_stats(start_key)->'intent_count' AS intent_count FROM crdb_internal.ranges_no_leases) WHERE intent_count != '0'
+~~~
+
+To force cleanup of intents, either of the following methods can be used:
+
+* Do a high-priority scan of the table, which will resolve intents as it runs. Note that this may abort any conflicting transactions that are currently running. If the table has indexes, these can be cleaned by changing `<table>` into `<table>@<index>`. The key ranges in the intent query above will output numeric table identifiers that can be entered as `[<tableid>]` instead of the table name.
```

Comment:
do we need to detail `[table_id]@[idx_id]`?

Reply:
May as well, update.